<launch>
    <arg name="lidar" default="false"/>
    <arg name="root_link" default="arm_base_link"/>
    <arg name="end_link" default="arm_tool_frame"/>
    <arg name="robot_name" default="$(env ROBOT_NAME)"/>
    <arg name="robot_type" value="kinova"/>

    <param name="robot_name" value="$(arg robot_name)" />
    <param name="root_link" value="$(arg root_link)" />
    <param name="end_link" value="$(arg end_link)" />
    <param name="lidar" value="$(arg lidar)"/>
    <param name="robot_type" value="$(arg robot_type)"/>

    <param name="dinova_fk_description" textfile="$(find dinova_control)/config/$(env ROBOT_NAME)_dinova.urdf"/>
    
    <node pkg="dinova_control" type="generate_fk.py" name="generate_fk" output="screen"/>

</launch>
